google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -750,6 +751,36 @@ public boolean isRefreshingChannel() {
       return primedTableIds;
     }
 
+    /**
+     * Enable latency based throttling for {@link BigtableDataClient#newBulkMutationBatcher(String)}
+     * with a target rpc latency. The number of allowed in-flight requests will be adjusted to reach
+     * the target bulk mutations rpc latency.
+     */
+    @BetaApi("Latency based throttling is not currently stable and may change in the future")
+    public Builder enableBatchMutationLatencyBasedThrottling(long targetRpcLatencyMs) {
+      bulkMutateRowsSettings.setLatencyBasedThrottling(true, targetRpcLatencyMs);
+      return this;
+    }
+
+    /**
+     * Disable latency based throttling for {@link
+     * BigtableDataClient#newBulkMutationBatcher(String)}.
+     */
+    @BetaApi("Latency based throttling is not currently stable and may change in the future")
+    public Builder disableBatchMutationLatencyBasedThrottling() {
+      bulkMutateRowsSettings.setLatencyBasedThrottling(false, null);
+      return this;
+    }
+
+    /**
+     * Gets if latency based throttling is enabled for {@link
+     * BigtableDataClient#newBulkMutationBatcher(String)}
+     */
+    @BetaApi("Latency based throttling is not currently stable and may change in the future")
+    public boolean isLatencyBasedThrottlingForBatchMutationEnabled() {
+      return bulkMutateRowsSettings.isLatencyBasedThrottlingEnabled();
+    }
+
```

Comment:
I dont think these provide much value, I think we can inline them in BigtableDataSettings